stubs/inertia/resources/js/Layouts/AppLayout.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div>
-        <jet-banner />
```

Comment:
I think you should revert any formatting changes, as they are not related to this PR.